### PR TITLE
Update lifecycle time periods

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: site/content/contribute/getting-started/inactive-contributions.md [hunk 1]
```diff
@@ -7,15 +7,15 @@ subsection: Getting Started
 
 This process describes how inactive contributions are managed at Mattermost, inspired by the [Kubernetes project](https://github.com/kubernetes/kubernetes):
 
-1. After 14 days of inactivity, a contribution becomes stale.
+1. After 10 days of inactivity, a contribution becomes stale.
  - If action is required from submitter, Community Coordinator asks if the team can help clarify previous feedback or provide guidance on next steps, and adds `lifecycle/1:stale` label to the contribution.
  - If action is required from reviewers, Community Coordinator asks reviewers to share feedback or help answer questions, and adds `lifecycle/1:stale` label to the contribution. The Coordinator will follow up with reviewers until a response is received.
 
-2. After another 14 days of inactivity, a contribution becomes inactive.
+2. After another 4 days of inactivity, a contribution becomes inactive.
```
Comment (hanzei) [inline site/content/contribute/getting-started/inactive-contributions.md]: this should stay at 14 days.

Comment (jasonblais) [reply site/content/contribute/getting-started/inactive-contributions.md]: Please see
 - https://community.mattermost.com/private-core/pl/odr33ttid3f4bgsq1kw8gjznih
 - https://community.mattermost.com/private-core/pl/cqsc8o34hfdkdn7qpe7nxcbrke

for context. If we keep it at 14 days, then the lifecycle will remain about the same.

Comment (hanzei) [reply site/content/contribute/getting-started/inactive-contributions.md]: Let's discuss this via DM first.

File: site/content/contribute/getting-started/inactive-contributions.md [hunk 2]
```diff
@@ -7,18 +7,18 @@ subsection: Getting Started
...
 
-3. After another 30 days of inactivity, a contribution becomes orphaned.
+3. After another 10 days of inactivity, a contribution becomes orphaned.
  - Community Coordinator notes that the contribution has been inactive for 60 days, thanks for the contribution and closes the contribution. They also add an `lifecycle/3:orphaned` label to the contribution, and adds an `Up For Grabs` label to the associated help wanted ticket, if appropriate.
 
 Special notes:
 
 1. If the contribution is inactive but shouldn't be closed, the Coordinator adds a `lifecycle/frozen` label to the contribution. An example of this is when a design decision is being discussed but no decision has been arrived at yet.
-2. In some instances, a contribution may be taken over by another contributor or staff member if it has become inactive. This should only occur once the contribution is in a `lifecycle/2:inactive` or `lifecycle/3:orphaned` state.
```
Comment (hanzei) [inline site/content/contribute/getting-started/inactive-contributions.md]: Why did you remove this? To me this unrelated the addition below.

File: site/content/contribute/getting-started/inactive-contributions.md [hunk 3]
```diff
@@ -7,18 +7,18 @@ subsection: Getting Started
 
 This process describes how inactive contributions are managed at Mattermost, inspired by the [Kubernetes project](https://github.com/kubernetes/kubernetes):
 
-1. After 14 days of inactivity, a contribution becomes stale.
+1. After 10 days of inactivity, a contribution becomes stale.
  - If action is required from submitter, Community Coordinator asks if the team can help clarify previous feedback or provide guidance on next steps, and adds `lifecycle/1:stale` label to the contribution.
  - If action is required from reviewers, Community Coordinator asks reviewers to share feedback or help answer questions, and adds `lifecycle/1:stale` label to the contribution. The Coordinator will follow up with reviewers until a response is received.
 
-2. After another 14 days of inactivity, a contribution becomes inactive.
+2. After another 10 days of inactivity, a contribution becomes inactive.
```
Comment (lieut-data) [inline site/content/contribute/getting-started/inactive-contributions.md]: The phrase "another 10" makes me do the math each time. Could we instead say:
```suggestion
2. After 20 days of inactivity, a contribution becomes inactive.
```

File: site/content/contribute/getting-started/inactive-contributions.md [hunk 4]
```diff
@@ -7,18 +7,18 @@ subsection: Getting Started
...
  - Community Coordinator asks the submitter if the team can help with questions. They acknowledge that after another 30 days of inactivity the contribution will be closed. They also add a `lifecycle/2:inactive` label to the contribution.
  - Note: Contributions should never become orphaned because of reviewers. The Coordinator will be responsible for receiving a response from the reviewers during the stale period, which may be that the maintainers aren't able to accept the contribution in its current form.
 
-3. After another 30 days of inactivity, a contribution becomes orphaned.
+3. After another 10 days of inactivity, a contribution becomes orphaned.
```
Comment (lieut-data) [inline site/content/contribute/getting-started/inactive-contributions.md]: As above, perhaps:
```suggestion
3. After 30 days of inactivity, a contribution becomes orphaned.
```

File: site/content/contribute/getting-started/inactive-contributions.md (hunk 2 above)
Comment (lieut-data) [reply site/content/contribute/getting-started/inactive-contributions.md]: I think I understand @jasonblais's thinking here in that the two paragraphs compete. Maybe we can reword to say:
```suggestion
2. Once the contribution reaches the `lifecycle/2:inactive` state, it is eligible to be assumed by another community member interested in working on the ticket.
3. Invalid PRs may be closed immediately without advancing through this lifecycle, especially if the contributor is unresponsive.
```

**Comment by hanzei** [comment]
@cpanato Would you please update the dates until `lifecycle/1:stale` is added in Mattermod according to this changes?

Basically 14 -> 10 days for `lifecycle/1:stale`
